site/src/pages/ManagementSettingsPage/OrganizationSettingsPage.tsx

```diff
@@ -32,37 +35,43 @@ const OrganizationSettingsPage: FC = () => {
     organizations && organizationName
       ? getOrganizationByName(organizations, organizationName)
       : undefined;
-  const permissionsQuery = useQuery(organizationPermissions(organization?.id));
+  const permissionsQuery = useQuery(organizationsPermissions(organizations));
 
-  if (!organizations) {
+  const permissions = permissionsQuery.data;
+  if (!organizations || !permissions) {
     return <Loader />;
   }
 
-  // Redirect /organizations => /organizations/default-org
+  // Redirect /organizations => /organizations/default-org, or if they cannot edit
+  // the default org, then the first org they can edit, if any.
   if (!organizationName) {
-    const defaultOrg = getOrganizationByDefault(organizations);
-    if (defaultOrg) {
-      return <Navigate to={`/organizations/${defaultOrg.name}`} replace />;
+    const editableOrg = organizations
+      .sort((a, b) => {
+        // Prefer default org (it may not be first).
+        if (a.is_default) {
+          return -1;
+        } else if (b.is_default) {
+          return 1;
+        }
+        return 0;
```

Comment:
little known, and kind of cursed, but...

```
> true - true
0
> true - false
1
> false - true
-1
```

so you could just do...

```suggestion
        // Prefer default org (it may not be first).
        return b.is_default - a.is_default
```

which is also technically more correct, because right now you're not properly checking the case where `a.is_default === true && b.is_default === true` (even though that should be impossible)

Reply:
Ahahahaha addition/substraction on booleans feels very on-brand for JavaScript lol

Good point on them both being true, will fix

Reply:
Uhh how bad is it if I have to disable a lint check to make this happen

Reply:
Turns out I am dumb and can just use some ternaries

Reply:
Thank you by the way, the way I did this was quite careless